### PR TITLE
Web3 config for actions

### PR DESCRIPTION
**Pull request by delfipolito**
image

### CODE REVIEW

File: src/sections/Function.js
Comment (brunitob) [inline src/sections/Function.js:1]: use more descriptive name, maybe something like:

- displayFunction
- renderFunction

File: src/sections/MoreActions.js [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
 import Table from '../components/Table/Table'
 import TableRow from '../components/Table/TableRow'
 import TableCell from '../components/Table/TableCell'
 import TableHeader from '../components/Table/TableHeader'
 import defaultAction from '../assets/default-action.svg'
 import useActionMetadata from '../hooks/useActionMetadata'
+import { shortenAddress } from '../utils/web3-utils'
 
 const MoreActions = ({ actions }) => {
   //TODO: remove two hardcoded items and replace them for real data
```
Comment (brunitob) [inline src/sections/MoreActions.js:14]:
```suggestion
```

File: src/utils/web3-utils.js [hunk 2]
```diff
@@ -15,6 +15,10 @@ export function shortenAddress(address, charsLength = 4) {
   )
 }
 
+export function convertWeiToGwei(wei) {
+  const gwei = wei / 1000000000
```
Comment (brunitob) [inline src/utils/web3-utils.js:19]:
```suggestion
  if (!wei) return 0
  const gwei = wei / 1000000000
```

**Review by brunitob — CHANGES_REQUESTED** [review]
🤝

File: src/sections/Function.js
Comment (delfipolito) [reply src/sections/Function.js:1]: How about ConfigParam?

**Review by brunitob — APPROVED** [review]
🎉 huge milestone

**Pull request merged**